ops/ansible/roles/bfd-server/tasks/install.yml

```diff
@@ -24,154 +24,17 @@
     mode: u=rwx,g=rx,o=rx
   become: true
 
-# TODO: This can be removed once we've fully migrated away from JBoss.
-# The first time we dpeloy the new Jetty-based server, we need to ensure that JBoss is stopped first (if it already exists).
-- name: Stop Existing BFD Server Service
-  service:
-    name: "{{ data_server_appserver_service }}"
-    state: stopped
-  become: true
-  ignore_errors: true
-
 - name: Copy BFD Server Launcher
   copy:
     src: "{{ data_server_launcher }}"
     dest: "{{ data_server_dir }}/{{ data_server_launcher | basename }}"
   become: true
-  notify:
-    - 'Restart BFD Server Service'
 
 - name: Copy BFD Server WAR
   copy:
     src: "{{ data_server_war }}"
     dest: "{{ data_server_dir }}/{{ data_server_war | basename }}"
   become: true
-  notify:
-    - 'Restart BFD Server Service'
-
-- name: Copy App Server Launcher Script
-  template:
-    src: bfd-server.sh.j2
-    dest: "{{ data_server_dir }}/bfd-server.sh"
-    owner: "{{ data_server_user }}"
-    group: "{{ data_server_user }}"
-    mode: u=rwx,g=rwx,o=rx
-  become: true
-  notify:
-    - 'Restart BFD Server Service'
-
-- name: Create BFD Server Service Definition
-  template:
-    src: bluebutton-appserver.service.j2
-    dest: "/etc/systemd/system/{{ data_server_appserver_service }}.service"
-    owner: root
-    group: root
-    mode: u=rw,g=r,o=r
-  become: true
-  notify:
-    - 'Enable BFD Server Service'
-    - 'Restart BFD Server Service'
-
-- name: Check for BFD Server SSL Keystore
-  stat:
-    path: "{{ data_server_dir }}/bluebutton-appserver-keystore.jks"
-  register: stat_appserver_keystore
-  changed_when: false
-  become: true
-
-- name: Create BFD Server SSL Keystore
-  command: "keytool -genkeypair -alias server {{ data_server_ssl_server_genkeypair_args }} -keystore {{ data_server_dir }}/bluebutton-appserver-keystore.jks -storepass changeit -keypass changeit"
-  when: stat_appserver_keystore.stat.isreg is not defined
-  become: true
-  become_user: "{{ data_server_user }}"
-  no_log: true  # Ensure that the key args aren't logged if this command fails.
-  notify:
-    - 'Restart BFD Server Service'
```

Comment:
Removing as this shouldn't be needed in CCS, we just copy the keystore contained within the repo after this task anyway.